metricbeat/module/dropwizard/collector/_meta/data.json

```diff
@@ -2,38 +2,19 @@
     "@timestamp": "2019-03-01T08:05:34.853Z",
     "dropwizard": {
         "testnamespace": {
-            "my_gauge": {},
-            "my_histogram": {
-                "count": 0,
-                "max": 0,
-                "mean": 0,
-                "min": 0,
-                "p50": 0,
-                "p75": 0,
-                "p95": 0,
-                "p98": 0,
-                "p99": 0,
-                "p999": 0,
-                "stddev": 0
+            "my_counter": {
```

Comment:
Why does it modify this one?

Reply:
Order of events depends on a hash of the whole event. For source files that generate multiple events is very likely that the order changes if anything in the content changes.